src/readii_2_roqc/utils/settings.py

```
@@ -0,0 +1,168 @@
+        assert check_setting_superset(CROP, crop)
+        # Get single crop value out of list format
+        crop = crop[0]
+    else:
+        crop = ""
+
+    resize = readii_config['IMAGE_TYPES']['resize']
+    if resize is None:
+        resize = []
+    elif isinstance(resize, list):
+        match len(resize):
+            case 1: resize = resize[0]
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)